documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -1609,100 +1609,35 @@ If the `continuous` parameter is set to `1`, the job pauses for the length of ti
-endif::community[]
-
-ifdef::product[]
-
-For details about deploying the {prodname} SQL Server connector, see the following topics:
-
-* xref:installing-debezium-sql-server-connectors[]
-* xref:debezium-sql-server-connector-configuration-example[]
-* xref:adding-sql-server-connector-configuration-to-kafka-connect[]
-* xref:descriptions-of-debezium-sql-server-connector-configuration-properties[]
-
-To deploy a {prodname} SQL Server connector, install the {prodname} SQL Server connector archive, configure the connector, and start the connector by adding its configuration to Kafka Connect.
-
-endif::product[]
-
-ifdef::product[]
-// Type: procedure
-// Title: Installing {prodname} SQL Server connectors
-[id="installing-debezium-sql-server-connectors"]
-==== Installing
-
-To install the SQL Server connector, follow the procedures in {LinkDebeziumInstallOpenShift}[{NameDebeziumInstallOpenShift}]. The main steps are:
-
-. Use link:https://access.redhat.com/products/red-hat-amq#streams[Red Hat AMQ Streams] to set up Apache Kafka and Kafka Connect on OpenShift. AMQ Streams offers operators and images that bring Kafka to OpenShift.
-
-. From a browser, open the link:https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?product=red.hat.integration&downloadType=distributions[Software Downloads page on the Red Hat Customer Portal],
-and download the {prodname} SQL Server connector.
-
-. Extract the connector files into your Kafka Connect environment.
-. Add the connector plug-in's parent directory to your Kafka Connect `plugin.path`, for example:
-+
-[source]
-----
-plugin.path=/kafka/connect
-----
-+
-The preceding example assumes that you extracted the {prodname} SQL Server connector to the `/kafka/connect/{prodname}-connector-sqlserver` path.
-
-. Restart your Kafka Connect process to ensure that it picks up the new JAR files.
-
-You also need to {LinkDebeziumUserGuide}#setting-up-sqlserver[set up SQL Server] to run a {prodname} connector.
-
-.Additional resources
-
-For more information about the deployment process, and deploying connectors with AMQ Streams, see the {prodname} installation guides.
+To deploy a {prodname} SQL Server connector, you install the {prodname} SQL Server connector archive, configure the connector, and start the connector by adding its configuration to Kafka Connect.
 
-* {LinkDebeziumInstallOpenShift}[{NameDebeziumInstallOpenShift}]
-* {LinkDebeziumInstallRHEL}[{NameDebeziumInstallRHEL}]
+.Prerequisites
+* link:https://zookeeper.apache.org/[Zookeeper], link:http://kafka.apache.org/[Kafka], and link:{link-kafka-docs}.html#connect[Kafka Connect] are installed.
+* SQL Server is installed and is {link-prefix}:{link-sqlserver-connector}#setting-up-sqlserver[set up to run the {prodname} connector].
```

Comment:
This one sounds a bit off, it's not exactly SQL Server that runs the Debezium connector. What exactly DYM here -- that SQL Server has CDC enabled and is ready to receive connections from the connector?

Reply:
> What exactly DYM here -- that SQL Server has CDC enabled and is ready to receive connections from the connector?

Yes, the Setting up SQL Server topic that's linked to is basically about enabling CDC, so I think that the point is that you've done that so that SQL Server can support now Debezium connections. 

A similar update is needed across all of the existing connector docs.

Reply:
Ok, that makes sense. So something like "ready to be used with the {prodname} connector" then would make sense.

Reply:
I replaced the original bullet text with the following:

 _SQL Server is installed, is {link-prefix}:{link-sqlserver-connector}#setting-up-sqlserver[configured for CDC], and is ready to be used with the {prodname} connector]._
 
 Good?

Reply:
Yepp, sounds good.